src/main/java/seedu/address/model/person/Family.java

```diff
@@ -0,0 +1,73 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents the family size of a person in the address book.
+ * Guarantees: family size is present and not null, and adheres to specific constraints.
+ */
+public class Family {
+
+    /** Message for constraints on family size. */
+    public static final String MESSAGE_CONSTRAINTS = "Family size should be at least one";
+
+    /** The family size. */
+    private Integer familySize;
+
+    /**
+     * Constructs a {@code Family} instance with the given family size.
+     *
+     * @param familySize The size of the family.
+     */
+    public Family(String familySize) {
+        requireNonNull(familySize);
+        this.familySize = Integer.valueOf(familySize);
+    }
+
+    /**
+     * Checks if the given family size is a valid value.
+     *
+     * @param familySize The family size to check.
+     * @return True if the family size is greater than or equal to zero, false otherwise.
+     */
+    public static boolean isValidFamily(String familySize) {
+        return Integer.parseInt(familySize) >= 0;
```

Comment:
Should this be >0 instead of >=0? Since the constraint is at least 1, not at least 0

Reply:
+1

Reply:
+1

Reply:
👍